src/update.sh

```
@@ -252,8 +300,8 @@ fn_update() {
         fi
   
         if [[ -d "./.git" ]] && [[ $FLAG_G == 1 ]]; then
-          # get the current version installed
-          new_version=$(get_package_version "$pkg_manager" "svelte")
+          # get the post-update version from package.json
+          new_version=$(get_package_version "svelte" "$pkg_json")
           newBannerColor "🏃 Running git commands ..." "magenta" "*"
           git add -A && git commit --message "Update Svelte to $new_version" && git push origin $(git branch --show-current)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)